projects/landing-page/src/App.tsx

```diff
@@ -156,7 +163,7 @@ const App: React.FunctionComponent = () => {
                   mt={2}
                 >{`await api1.rpc.chain.subscribeNewHeads((lastHeader) => {`}</Box>
                 <Box pl={3}>{`console.log(lastHeader.hash);`}</Box>
-                <Box>{`);`}</Box>
+                <Box>{`});`}</Box>
```

Comment:
The missing } is already fixed

Reply:
sorry, merged `main` to my branch.

but the previous fix hasn't landed to https://paritytech.github.io/substrate-connect/ yet.
it still shows like this to me:
 ![](https://i.imgur.com/pMlkpF0.png)

Reply:
That is true. I will deploy soon-ish